src/main/java/fi/okm/jod/yksilo/entity/Osaaminen.java

```diff
@@ -45,6 +38,13 @@ public class Osaaminen {
   @BatchSize(size = 1000)
   private Map<Kieli, Kaannos> kaannos;
 
+  @ManyToMany(fetch = FetchType.LAZY)
+  @JoinTable(
+      name = "yksilo_osaamis_kiinnostukset",
+      joinColumns = @JoinColumn(name = "osaamis_kiinnostukset_id"),
+      inverseJoinColumns = @JoinColumn(name = "yksilo_id"))
+  private Set<Yksilo> kiinnostuneet;
```

Comment:
This is incorrect. Osaaminen must not have a back-reference to yksilo, even if it is lazy. This collection, If ever populated, could load ALL users from the database.